AllReadyApp/Web-App/AllReady/Features/Notifications/NotifyAdminForTaskSignupStatusChangeHandler.cs

```diff
@@ -43,7 +43,7 @@ public NotifyAdminForTaskSignupStatusChangeHandler(AllReadyContext context, IMed
             {
                 var link = $"View event: http://{_options.Value.SiteBaseUrl}/Admin/Task/Details/{taskSignup.Task.Id}";
 
-                var subject = !string.IsNullOrEmpty(volunteer.FirstName) && !string.IsNullOrEmpty(volunteer.LastName)
+                var subject = volunteer.FirstName != null && volunteer.LastName != null

```

Comment:
@mgmccarthy ok, I thought we should go back to what was initially. I will change it in a minute.